.travis.yml

```diff
@@ -106,3 +100,15 @@ jobs:
         edge: true
         # Run `semantic-release`
         script: 'npx semantic-release@15.14'
+
+# Notification options: `always`, `never` or `change`
+notifications:
+  webhooks:
+    if: 'repo = saltstack-formulas/suricata-formula'
+    urls:
+      - https://saltstack-formulas.zulipchat.com/api/v1/external/travis?api_key=HsIq3o5QmLxdnVCKF9is0FUIpkpAY79P&stream=CI&topic=saltstack-formulas%2Fsuricata-formula&ignore_pull_requests=true
+    on_success: always  # default: always
+    on_failure: always  # default: always
+    on_start: always    # default: never
+    on_cancel: always   # default: always
+    on_error: always    # default: always
```

Comment:
This is enabled for all formulas -- won't do anything here until this formula is transferred/forked into the organisation.

Reply:
Okay, I think this was removed when running the template prep script https://github.com/saltstack-formulas/template-formula/blob/master/.travis.yml#L144-L157. Thanks for adding it back.